### PR TITLE
 diodon: add at 1.13.0 

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- Add at 1.13.0
- Resolves https://github.com/getsolus/packages/issues/3350
- Resolves https://github.com/getsolus/packages/issues/3349

**Test Plan**
- Installed, copied some text and images and checked it was added.

**Checklist**

- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/d/diodon/package.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+source     :
+    - https://github.com/diodon-dev/diodon/archive/refs/tags/1.13.0.tar.gz : 387e78e24dfcac5a73c65ff758c66894a712f7d10754b4ffcfdc11590a9e0a10
+homepage   : https://github.com/diodon-dev/diodon
+license    : GPL-2.0-or-later
+component  : desktop
+summary    : Aiming to be the best integrated clipboard manager for the Unity desktop.
+description: |
+    Aiming to be the best integrated clipboard manager for the Unity desktop.
+replaces   :
+   - clipit
```
Comment (malfisya) [inline packages/d/diodon/package.yml:13]: Add :
```
 - dbginfo : clipit-dbginfo

```

File: packages/d/diodon/package.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+release    : 1
+source     :
+    - https://github.com/diodon-dev/diodon/archive/refs/tags/1.13.0.tar.gz : 387e78e24dfcac5a73c65ff758c66894a712f7d10754b4ffcfdc11590a9e0a10
+homepage   : https://github.com/diodon-dev/diodon
+license    : GPL-2.0-or-later
+component  : desktop
+summary    : Aiming to be the best integrated clipboard manager for the Unity desktop.
+description: |
+    Aiming to be the best integrated clipboard manager for the Unity desktop.
+replaces   :
```
Comment (malfisya) [inline packages/d/diodon/package.yml:12]: I prefer replace key is at the bottom

File: packages/d/diodon/package.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+version    : 1.13.0
+release    : 1
+source     :
+    - https://github.com/diodon-dev/diodon/archive/refs/tags/1.13.0.tar.gz : 387e78e24dfcac5a73c65ff758c66894a712f7d10754b4ffcfdc11590a9e0a10
+homepage   : https://github.com/diodon-dev/diodon
+license    : GPL-2.0-or-later
+component  : desktop
+summary    : Aiming to be the best integrated clipboard manager for the Unity desktop.
+description: |
+    Aiming to be the best integrated clipboard manager for the Unity desktop.
```
Comment (malfisya) [inline packages/d/diodon/package.yml:11]: Is this only target Unity? We don't have unity

File: packages/d/diodon/pspec_x86_64.xml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>diodon</Name>
+        <Summary xml:lang="en">Aiming to be the best integrated clipboard manager for the Unity desktop.</Summary>
+        <Description xml:lang="en">Aiming to be the best integrated clipboard manager for the Unity desktop.
+</Description>
+        <PartOf>desktop</PartOf>
+        <Files>
+            <Path fileType="config">/etc/xdg/autostart/diodon-autostart.desktop</Path>
```
Comment (malfisya) [inline packages/d/diodon/pspec_x86_64.xml:23]: No shipping file outside of `/usr`

**Review by malfisya — CHANGES_REQUESTED** [review]
A few things. Deprecation also needs rebase

File: packages/d/diodon/package.yml (hunk 3 above)
Comment (androidnisse) [reply packages/d/diodon/package.yml:11]: No, that is from their description. It is in Debian and they don't have Unity.

Comment (malfisya) [reply packages/d/diodon/package.yml:11]: Does debian describe it as such? It was potentially confusing for user. Maybe make it more generic

Comment (androidnisse) [reply packages/d/diodon/package.yml:11]: Kinda they mention gnome as well.
https://packages.debian.org/bookworm/diodon

Comment (malfisya) [reply packages/d/diodon/package.yml:11]: Lets use the debian description. 
```
summary : GTK+ Clipboard manager
description : 
      blah blah blah
```

Change the metainfo too

**Review by malfisya — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**